docs/osf/setup.rst

```diff
@@ -79,7 +79,7 @@ In your text editor, add the following line to your .bash_profile
 
     ::
 
-        PATH=$PATH:/usr/local/bin
+        PATH=$PATH:/usr/local/sbin
```

Comment:
Most packages installed with homebrew go in `/usr/local/bin`, only a few go in `/usr/local/sbin`. Both should be added to the path:

`PATH=/usr/local/bin:/usr/local/sbin:$PATH`

Also, putting `$PATH` at the end means that anything installed with homebrew takes precedence over system-installed packages. This is especially useful for installing Python via homebrew.